packages/common/src/utils/formatting.ts

```diff
@@ -270,7 +284,7 @@ export function formatFieldValue(
         case MetricType.COUNT:
         case MetricType.COUNT_DISTINCT:
         case MetricType.SUM:
-            return formatValue(value, { format, round, compact });
+            return formatValue(value, formatOpts);
```

Comment:
let's bring the previous change back: 

```
return formatValue(value, { format, round, compact });
```

and remove the `isAdditionalMetric` check 👍

Reply:
Done as well 👌